solutions/usaco-615.mdx

```diff
@@ -2,16 +2,49 @@
 id: usaco-615
 source: USACO Bronze 2016 February
 title: Milk Pails
-author: Danh Ta Chi Thanh
+author: Ryan Chou, Danh Ta Chi Thanh
 ---
 
 [Official Analysis (Java)](http://www.usaco.org/current/data/sol_pails_bronze_feb16.html)
 
+## Implementation
+
 <LanguageSection>
+	
+<PySection>
 
-<CPPSection>
+```python
 
-## Implementation
+
```

Comment:
could you delete the blank lines before the code? (lines 17 and 18)
other than that, it looks good!

Reply:
sure!

Reply:
They've been deleted!